### PR TITLE
[#13] Fix deployment script problems

### PR DESCRIPTION
**Pull request by cbcunc**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,15 +3,17 @@ MAINTAINER Michael J. Stealey <mjstealey@gmail.com>
...
     openssh-client \
     openssh-server \
     supervisor \
     rsync
 
-COPY . /tmp
-RUN cp /tmp/requirements.txt /requirements.txt
+# For local development?
+# COPY . /tmp
+# RUN cp /tmp/requirements.txt /requirements.txt
```
Comment (hyi) [inline Dockerfile]: Three commented lines above can be deleted.

File: pagemill/init-web [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 
 sleep 3s
-/usr/sbin/sshd
-/usr/bin/env bash run-server
\ No newline at end of file
+# For local development?
+# /usr/sbin/sshd
```
Comment (hyi) [inline pagemill/init-web]: Two commented lines above can be deleted.

File: pmctl [hunk 3]
```diff
@@ -52,6 +61,8 @@ start_nginx() {
     echo "*** START NGINX ***"
     if [[ ! -n "$(docker-compose ps | grep nginx | grep Up)" ]]; then
         echo "*** INFO: nginx is not running, waiting ***"
```
Comment (hyi) [inline pmctl]: echo message could be changed to replacing ```waiting``` with ```starting``` to align with the action better.

File: pmctl [hunk 4]
```diff
@@ -124,7 +135,14 @@ build_pagemill() {
...
 run_pagemill() {
-    echo "*** RUN ***"
+    echo "*** RUN ***" 
+    # Create the media path before the container is run
+    cd ${PAGEMILL_PATH}
+    if [ ! -d "static/media" ]; then
+        mkdir -p static/media;
+    fi
+    cp -r media_files/media static/
+    cd -
```
Comment (hyi) [inline pmctl]: Are these statements for creating static/media directory and copying media files over already done as part of main pmctl code line as shown above? If so, they will not be needed here.

**Comment by devincowan** [comment]
closing as no longer relevant

**Pull request closed without merging**